### DISCUSSION TITLE
How to do git revert?

### DISCUSSION THREAD

**Discussion by fullstackdev48** [fullstackdev48/shopify-template-ruby#9] [Q&A]
Please explain how to revert the commit.

**Comment by outdoorSpirit** [comment]
If you need to revert a commit, you can use the `git revert` command followed by the hash of the commit you wish to undo. This will create a new commit that applies the inverse of the changes made in the specified commit. To find the commit hash, you can use `git log` to display a list of recent commits. After running the revert command, Git will prompt you to enter a commit message for the new commit. Finally, you can push the new commit to the remote repository to reflect the reversion there as well.
